src/components/StudentHome.jsx

```diff
@@ -1,16 +1,28 @@
 import React from 'react';
+import { Row, Col, Alert, Glyphicon } from 'react-bootstrap';
 import WelcomeMessage from './WelcomeMessage';
 import PlanOfStudy from './PlanOfStudy';
+import NavPanel from './NavPanel';
 
 // The wrapper for all components on a student home page
 const StudentHome = () => (
-  <div>
-    <h2>
-      Student Home
-    </h2>
-    <WelcomeMessage student="Ariana Olson" />
-    <PlanOfStudy />
-  </div>
+  <Row>
```

Comment:
oh yeah, that was kind of a nightmare. It's to make it behave properly on different screen sizes, I can try to put some explanatory stuff in?